aws-params.yaml

```diff
@@ -5,4 +5,5 @@ metaphlan_db: "s3://biobakery-databases/metaphlan_databases/"
 humann_bowtie_db: "s3://biobakery-databases/humann_databases/chocophlan"
 humann_protein_db: "s3://biobakery-databases/humann_databases/uniref"
 humann_utility_db: "s3://biobakery-databases/humann_databases/utility_mapping"
-filepattern: "*_L00{1,2,3,4}_R{1,2}_001.fastq.gz"
\ No newline at end of file
+filepattern: "*.fastq.gz"
```

Comment:
This is the pattern that I always have to change. Im thinking here that his specific string should be a command-line argument. In this case, for eample, it's easy, because it's just `"*.fastq"`, but I had to do more complicated stuff in the past, especially with paired data.

Reply:
You can always pass arguments to the command line to override stuff in the params file - `--filepattern="*.fastq"`

Reply:
But also, these params files are meant to be templates, not the thing you use in the end :shrug:

Reply:
> You can always pass arguments to the command line to override stuff in the params file - `--filepattern="*.fastq"`

I was today years wold when I found that out. The more you know/